In [5]:
import numpy as np

#### 1. Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16. Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95, если выборочная средняя M = 80, а объем выборки n = 256

$\overline{X} \pm z_{a/2} * \frac{\sigma}{\sqrt{n}} = 80 \pm 1.96 * \frac{16}{16} = [78.04;81.96]$


#### 2. В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, получены опытные данные:
6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1
Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей,
оценить истинное значение величины X при помощи доверительного интервала, покрывающего это
значение с доверительной вероятностью 0,95.

In [6]:
a = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])
M = round(a.sum()/len(a),2)
M

6.59

In [7]:
S = np.sqrt(((a-M)**2).sum()/(len(a)-1))
S

0.4508017549014448

$T_{1, 2} = \overline{X} \pm t_{a/2} * \frac{\sigma}{\sqrt{n}}$

In [8]:
M + 2.262 * S/np.sqrt(10)

6.912461744087574

In [9]:
M - 2.262 * S/np.sqrt(10)

6.2675382559124255

Доверительный интервал [6.268 - 6.912]

#### 3. Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм. Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр оказался равным 17.5 мм, а дисперсия известна и равна 4 кв.мм.

H0: диаметр = 17мм, $\mu = \mu_{0}$

H1: диаметр = 17.5мм, $\mu > \mu_{0}$

In [10]:
Zн = (17.5 - 17) / (2/10)
Zн

2.5

H1 верна, тк. Zт 1.645 < Zн

4. Продавец утверждает, что средний вес пачки печенья составляет 200 г.
Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:
202, 203, 199, 197, 195, 201, 200, 204, 194, 190.
Известно, что их веса распределены нормально.
Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

In [11]:
b = np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])

In [12]:
Mb = round(b.sum()/len(b),2)
Mb

198.5

In [13]:
Sb = np.sqrt(((b-Mb)**2).sum()/(len(b)-1))
Sb

4.453463071962462

In [15]:
Tp = (Mb-200)/ (Sb/np.sqrt(10))
Tp

-1.0651074037450896

-3.25 < -1.065 < 3.25 - при вероятности 99% утверждение верно

In [34]:
Mb+3.25 * Sb/np.sqrt(10)

203.0770032044268

In [35]:
Mb-3.25 * Sb/np.sqrt(10)

193.9229967955732

Доверительный интервал при вероятности 99% от 193.92 до 203.08